package.json

```diff
@@ -141,6 +141,6 @@
   },
   "dependencies": {
     "@types/temp": "^0.8.34",
-    "aws-sdk": "^2.727.1"
+    "aws-sdk": "^2.789.0"
```

Comment:
hm, these should probably be in devDependencies.

Reply:
@nornagon should i create a PR to move this to devDep?
Would also updating it at the same time be OK or not?